Cron_Philip/Assignments/oneController/oneController.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html ng-app="app">
+<head>
+    <meta charset="utf-8">
+    <title>Controller and Directives</title>
+
+    <!-- bootstrap -->
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+    <!-- angular -->
+
+    <script src = 'bower_components/angular/angular.js'></script>
+    <script>
+
+    var app = angular.module('app', []);
+    app.controller('foodController', ['$scope', function($scope) {
+
+        $scope.food = [];
+
+        $scope.addFood = function(foodInput) {
+            if (foodInput && foodInput.length >= 1) {
+                $scope.food.push(foodInput);
+                $scope.foodInput = '';
+            }
+        }
+
+        $scope.printToConsole = function() {
+            console.log("This form was submitted!");
+        }
+
+    }]);
+
+
+    </script>
+</head>
+<body>
+
+    <div class="container">
+        <div ng-controller = "foodController">
+            <h3>Welcome to the controller</h3>
+            <hr>
+            <br>
+            <div class="col-md-6">
+                <form ng-submit="printToConsole()">
+                    Favorite Food: <input type="text" ng-model="foodInput" placeholder="Food">
+                    <button class="btn btn-primary" ng-click = "addFood(foodInput)">Add Food</button>
```

Comment:
Good. But since foodInput is ng-model'd, no need to pass it in as a parameter to addFood(); It'll already exist... so long as the user has entered something into the input box.